src/Components/src/Microsoft.AspNetCore.Components.Server/Builder/FrameworkFilesProvider.cs

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Microsoft.Extensions.FileProviders;
+using Microsoft.Extensions.Primitives;
+
+namespace Microsoft.AspNetCore.Builder
+{
+    internal class FrameworkFilesProvider : IFileProvider
```

Comment:
You're right. I've updated the code to remove `FrameworkFilesProvider` and use `EmbeddedFileProvider`.

Previously I was trying to unify how the `components.server.js` file and the `blazor.boot.json` file were served. But since `blazor.boot.json` is temporary it doesn't make sense to orient the design around that.

> If this is a temporary workaround for some stuff

Serving a stub file for `blazor.boot.json` is a temporary workaround, but serving `components.server.js` from an embedded resource will hopefully remain.